apps/cancel-request/sections/summary-data-sections.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
       step: '/cancel-request-who-representing',
       field: 'cnc-who-is-representing'
     },
+    {
+      step: '/cancel-request-application',
+      field: 'cnc-reason-for-application'
```

Comment:
summary page label for this field is different. please add a entry in pages.json 
```
 "confirm": {
    "fields": {
```

Reply:
Right, thanks for highlighting this. I was aware that the field name is different and I thought we will have to handle this in `parse` method :D . Will update this in the translation

Reply:
Updated.